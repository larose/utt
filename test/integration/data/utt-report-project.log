2018-08-20 08:00 hello
2018-08-20 08:06 project_1: task_1
2018-08-20 08:12 project_2: task_1
2018-08-20 08:18 project_1: task_2
2018-08-20 08:24 project_2: task_2

2018-08-21 08:00 hello
2018-08-21 08:06 project_1: task_1
2018-08-21 08:12 project_2: task_1
2018-08-21 08:18 project_1: task_2
2018-08-21 08:24 project_2: task_2
2018-08-21 08:30 project_1: task_3
2018-08-21 08:36 project_2: task_3
