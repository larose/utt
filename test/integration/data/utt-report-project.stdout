
------------------------ Monday, Aug 20, 2018 (week 34) ------------------------

Working Time: 0h12
Break   Time: 0h00

----------------------------------- Projects -----------------------------------

(0h12) project_1: task_1, task_2

---------------------------------- Activities ----------------------------------

(0h06) project_1: task_1
(0h06) project_1: task_2


----------------------------------- Details ------------------------------------

(0h06) 08:00-08:06 project_1: task_1
(0h06) 08:12-08:18 project_1: task_2

